sae_lens/sae.py

```diff
@@ -61,6 +61,7 @@ class SAEConfig:
     sae_lens_training_version: Optional[str]
     activation_fn_kwargs: dict[str, Any] = field(default_factory=dict)
     neuronpedia_id: Optional[str] = None
+    grid_wise: bool = False
```

Comment:
Why do we need this if it's in `DRCSAERunnerConfig` ?

Reply:
SAELens uses this config for inference and `LanguageModelSAERunnerConfig` for training. `DRCSAERunnerConfig` is used for training and this `SAEConfig` class also needs `grid_wise` for running inference.